wikitexthtml/render/section.py

```diff
@@ -163,8 +163,7 @@ def replace(instance: WikiTextHtml, wikitext: wikitextparser.WikiText):
             title = section.title.strip()
 
             content = f"<h{section.level}>"
-            content += f'<a class="anchor" id="{slug}" href="#{slug}"></a>'
-            content += f'<a name="{slug}" id="{slug}"></a>'
+            content += f'<a class="anchor" href="#{slug}" name="{slug}"></a>'
```

Comment:
Either we should merge those and remove `name`, or keep them as 2 entries. The second `<a>` is the anchor, but HTML5 deprecated that, so I am fine with both. But either one, not combination :)